runtime/drivers/bigquery/bigquery.go

```diff
@@ -38,7 +38,7 @@ var spec = drivers.Spec{
 		{
 			Key:         "project_id",
 			Type:        drivers.StringPropertyType,
-			Required:    true,
+			Required:    false,
```

Comment:
This just makes the form easier for a user to fill if the project id from a query is a billing project id.
But if there is an SDK issue then the change might be reverted so that client project is mandatory for all cases

Reply:
Yeah initially we kept is as optional but some users complained that they find it weird that its marked as optional but later complains that we need to add `project_id` so we marked this as mandatory.